### PR TITLE
Add DNS support

### LINKED ISSUE #9: Configure kube-dns
Add DNS support.

### PR DESCRIPTION
**Pull request by kosyfrances**
This PR adds DNS configuration to the cluster.

Depends on #8
Fixes #9

### CODE REVIEW

**Comment by schu** [comment]
> As with the other pull request, there's no dependency on #8...

But I haven't checked if the kube-dns manifest as-is is compatible with Kubernetes 1.8.

**Comment by kosyfrances** [comment]
@schu we probably should not worry about checking this for k8s 1.8 IMO.

**Comment by schu** [comment]
> we probably should not worry about checking this for k8s 1.8 IMO.

Yes, 1.8 is EOL.

File: scripts/setup [hunk 1]
```diff
@@ -169,3 +173,6 @@ cat >>/etc/hosts <<DNS_IPS
 DNS_IPS
 EOF
 done
+
+# Add kube-dns
+kubectl create -f manifests/kube-dns.yaml
```
Comment (schu) [inline scripts/setup]: Should also be mentioned in the README to be on par.

**Comment by kosyfrances** [comment]
@schu this is ready for another review please :)

**Comment by schu** [comment]
ae99db5 seems to be a fixup for eb136d8 and should be squashed I think, otherwise looks good to me from a cursory look!

**Comment by kosyfrances** [comment]
fixed.

**Pull request merged**